### PR TITLE
Add Linux Support to Battery Plus

### PR DESCRIPTION
**Pull request by mhadaily**
moved from here https://github.com/fluttercommunity/battery_plus/pull/5

### CODE REVIEW

**Comment by miquelbeltran** [comment]
run it on my machine and worked as expected :+1:

**Comment by jpnurmi** [comment]
Rebased and resolved conflicts after merging #2 for macOS

**Comment by miquelbeltran** [comment]
hi! whats stopping us from merging this one?

**Review by jpnurmi — COMMENTED** [review]
> hi! whats stopping us from merging this one?

This will be merged soon. :) We have just reviewed the code with @mhadaily.

File: packages/battery_plus_linux/pubspec.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+name: battery_plus_linux
+description: Linux implementation of the battery_plus plugin
+homepage: https://plus.fluttercommunity.dev/
+repository: https://github.com/fluttercommunity/plus_plugins/tree/main/packages/
+version: 0.1.0
+
+environment:
+  sdk: ">=2.7.0 <3.0.0"
+  flutter: ">=1.17.0 <2.0.0"
```
[the remark this answers is not in the document]
Comment (jpnurmi) [reply packages/battery_plus_linux/pubspec.yaml:9]: hmm, the problem went away after rebasing on latest origin/main

**Pull request merged**